src/actor.js

```diff
@@ -764,3 +764,40 @@ export const getApifyProxyUrl = (options = {}) => {
 
     return `http://${username}:${password}@${hostname}:${port}`;
 };
+
+/**
+ *
+ * Creates adhoc webhook for current actor run.
+ * Webkooks are supported only for runs on Apify cloud.
+ *
+ * @param eventTypes {String[]} - Array of event types, which you can set for actor run, see
+ * the <a href="https://apify.com/docs/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.
+ * @param requestUrl {String} - URL which will be requested using HTTP POST request, when actor run will be in specific event type.
+ *
+ * @return {Promise<Webhook|undefined>}
+ *
+ * @memberof module:Apify
+ * @function
+ * @name createAdHocWebhook
+ */
+export const createAdHocWebhook = async (eventTypes, requestUrl) => {
```

Comment:
This name is not good, as we discussed before. People don't know what is "ad-hoc webhook". I'm in favor of calling this just:

```js
Apify.webhook({ eventTypes, requestUrl, ... });
```

And use object parameter

Reply:
@mtrunkat @mnmkng what do you think?

Reply:
Definitely, we need more actions later.